app/src/main/java/com/bogdan/codeforceswatcher/features/news/cells/PostPinned.kt

```diff
@@ -0,0 +1,260 @@
+package com.bogdan.codeforceswatcher.features.news.cells
+
+import android.graphics.PorterDuff
+import android.graphics.PorterDuffXfermode
+import android.graphics.Typeface
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.*
+import androidx.compose.ui.graphics.drawscope.drawIntoCanvas
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.bogdan.codeforceswatcher.R
+import com.bogdan.codeforceswatcher.components.compose.buttons.CommonButton
+import com.bogdan.codeforceswatcher.components.compose.theme.AlgoismeTheme
+import com.bogdan.codeforceswatcher.features.news.models.NewsItem
+
+@Composable
+fun PostPinnedView(
```

Comment:
`modifier` is a "must" for non-private composables.

Reply:
Anywhere? Even if I don't have any modifiers to pass?

Reply:
Yes, it's a convention. And you should use it for the top-most composable inside.